lib/auth/auth_with_roles.go

```diff
@@ -4542,7 +4542,7 @@ func (a *ServerWithRoles) GetSAMLIdPSession(ctx context.Context, req types.GetSA
 	}
 	// Users can only fetch their own SAML IdP sessions.
```

Comment:
Update the comment too?
```suggestion
	// Users can only fetch their own web sessions.
```